lib/presentation/shared/form/util/image_cropper_dialog.dart

```diff
@@ -0,0 +1,131 @@
+import 'dart:typed_data';
+
+import 'package:collaction_cms/presentation/shared/buttons/buttons.dart';
+import 'package:collaction_cms/presentation/theme/constants.dart';
+import 'package:crop_your_image/crop_your_image.dart';
+import 'package:flutter/material.dart';
+
+Future<Uint8List?> showImageCropperModal(
+  BuildContext context,
+  Size targetSize,
+  Uint8List image, {
+  Function? callback,
+  double horizontalPadding = 24,
+  double cropperWidth = 400,
+  double cropperMargin = 30,
+}) async {
+  CropController controller = CropController();
+  double aspectRatio = targetSize.width / targetSize.height;
```

Comment:
Size already has a getter called `aspectRatio`
https://api.flutter.dev/flutter/dart-ui/Size/aspectRatio.html

But it might be confusing to pass a ratio as a Size. People might expect to pass a width and height like Size(200, 50) etc. Might be better to create a class like `Ratio()` to avoid confusions.

Reply:
Did not know about the getter, will change that.

As for use of Size, that was left over from before @laschnik proposed the use of standard 16:9. Perhaps the simplest approach would be to change it to  `double aspectRatio = 16 / 9`?